src/Catalyst.TestUtils/FileSystemBasedTest.cs

```diff
@@ -123,8 +125,17 @@ protected virtual void Dispose(bool disposing)
             var regex = new Regex(CurrentTestName + @"_(?<timestamp>[\d]{16})");
 
             var oldDirectories = _testDirectory.Parent.EnumerateDirectories()
-               .Where(d => regex.IsMatch(d.Name)
-                 && string.CompareOrdinal(d.Name, _testDirectory.Name) == -1)
+               .Where(d =>
+                {
+                    var matches = regex.Matches(d.Name);
+                    if (matches.Count == 0) return false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4375968)